streetcrm/admin.py

```diff
@@ -530,6 +534,12 @@ def export_search(request, form, advanced, search_query):
                 elif col['column'] == 'major_action_id' and result.__dict__[col['column']] != None:
                     this_action = models.Event.objects.get(id=result.__dict__[col['column']])
                     new_row.append(this_action.name)
+                elif col['column'] == 'leadership':
+                    # If there are any leadership stages, convert them to a list so that the database
+                    # isn't queried again and pull the most recent
+                    if result.tracked_growth.all():
+                        stages = sorted([growth for growth in result.tracked_growth.all()], key=lambda g: g.date_reached)
+                        new_row.append(stages[-1].name)
```

Comment:
This needs one more step to work for me:
```
-                        new_row.append(stages[-1].name)
+                        new_row.append(stages[-1].stage.name)
```
Otherwise exporting the results gets:
```
AttributeError at /search/
'LeadershipGrowth' object has no attribute 'name'
```

Is that true for you too, @pjsier?

Reply:
@cecilia-donnelly ah I must have changed it the growth model but not updated the attribute, good catch! Fixed it now